File: 240307/시험 통과 여부 확인하기 2/verify-test-passed-2.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
    int n;
    cin>>n;
    int cnt=0;

    for(int i=0;i<n;i++){
        int arr[4];
        int sum=0;

        for(int j=0;j<4;j++){
            cin>>arr[j];
            sum+=arr[j];
        }
        double avg=(double)sum/4;

        if(avg>=60) {cout<<"pass"<<"\n";
        cnt++;}
        else cout<<"fail"<<"\n";
    }
    cout<<cnt;
    return 0;
}

File: 240527/시험 통과 여부 확인하기 2/verify-test-passed-2.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int cnt = 0;
        int n = sc.nextInt();

        for(int i = 0; i < n; i++) {
            int sum = 0;
            int[] arr = new int[4];
            
            for(int j = 0; j < 4; j++) {
                arr[j] = sc.nextInt();
            }
            
            for(int j = 0; j < 4; j++) {
                sum += arr[j];
            }

            double avg = (double)sum / 4;

            if(avg >= 60) {
                System.out.println("pass");
                cnt++;
            } else {
                System.out.println("fail");
            }
        }

        System.out.println(cnt);
    }
}